tests/cases/fourslash/inlayHintsShouldWork45.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference path="fourslash.ts" />
+
+// @allowJs: true
+// @checkJs: true
+
+// @Filename: /a.js
+//// module.exports.a = 1
+
+// @Filename: /b.js
+//// const a = require('./a');
+
+goTo.file('/b.js')
+const markers = test.markers();
+verify.getInlayHints([], undefined, {

```

Comment:
What does this test do?

Reply:
It's means 'No hints provided'. (as well as the below one)

Reply:
Is that meaningful without any markers in the file?

Reply:
Well... I missed that...😂